controllers/review-controllers.js

```diff
@@ -35,3 +35,15 @@ exports.getAllReviews = (request, response) => {
         response.status(200).send({ reviews: reviews })
     })
 }
+
+//A controller for fetching a comment object based on a review_id
+exports.getReviewComment = (request, response, next) => {
+    const {review_id} = request.params;
+    selectReviewComment(review_id).then((comment) => {
+        console.log(Object.keys(comment));
+        response.status(200).send({ comment });
+    })
+    .catch((error) => {
+        next(error);
+    })
+}
```

Comment:
make sure to remove console logs before merging to main!

Reply:
have done.